test/components/plots/gridPlotTests.ts

```diff
@@ -100,13 +100,13 @@ describe("Plots", () => {
       ];
       dataset.data(data);
       var cells = (<any> gridPlot)._renderArea.selectAll("rect")[0];
-      assert.equal(cells.length, data.length);
+      assert.strictEqual(cells.length, data.length);
       for (var i = 0; i < cells.length; i++) {
         var cell = d3.select(cells[i]);
-        assert.equal(cell.attr("x"), i * CELL_WIDTH, "Cell x coord is correct");
-        assert.equal(cell.attr("y"), i * CELL_HEIGHT, "Cell y coord is correct");
-        assert.equal(cell.attr("width"), CELL_WIDTH, "Cell width is correct");
-        assert.equal(cell.attr("height"), CELL_HEIGHT, "Cell height is correct");
+        assert.strictEqual(+cell.attr("x"), i * CELL_WIDTH, "Cell x coord is correct");
+        assert.strictEqual(+cell.attr("y"), i * CELL_HEIGHT, "Cell y coord is correct");
+        assert.strictEqual(+cell.attr("width"), CELL_WIDTH, "Cell width is correct");
```

Comment:
You have coercing here, hence unconsistent. Did not really get your argument, why was this out of the scope of the PR. Stringify the right handside